src/merge-folders-spec.js

```diff
@@ -0,0 +1,183 @@
+const path = require('path')
+const assert = require('assert')
+
+const isFunction = x => typeof x === 'function'
```

Comment:
I didn't want to load any libraries just to use one function from it, especially if the function is as simple as this. If I would do more type checks, then I would definietly use check-more-types.